### DISCUSSION TITLE
LanceDB Vector store node

### DISCUSSION THREAD

**Discussion by jclaveau** [FlowiseAI/Flowise#3446] [Ideas]
Presently we can use Redis, Pg, Milvus, Pinocone... as vector db which all require a third-party service to run.

[LanceDB](https://github.com/lancedb/lancedb) is an open source vector db stored in a local file and already [supported by LangChain](https://python.langchain.com/docs/integrations/vectorstores/lancedb/). 

The SQLite of vector dbs in a nutshell

Having a node to use it for embeddings would allow Flowise to run a lot of flows `requiring just a docker volume` and no other service

I may find the time to implement it if it gets some traction

**Comment by erlebach** [comment]
Any news on this? Any thoughts from the Flowise development team?

**Reply by jclaveau** [reply]
Personally, I switched to PGVector which is consistent with my stack and easy to query so I won't have enough need to implement it.

I'm also questioning the use of Flowise for other subjects than prototyping due to my need of versioned graphs btw
